_posts/2021-11-02-6.0.0-rs-release.md

```diff
@@ -0,0 +1,152 @@
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+We recently released the 6.0.0 Rust version of [Apache Arrow](https://arrow.apache.org/) which 
+coincides with the [Arrow 6.0.0 release](https://arrow.apache.org/release/6.0.0.html). This 
+post highlights some of the improvements in the Rust implementation. The full changelog can be 
+found [here](https://github.com/apache/arrow-rs/blob/6.0.0/CHANGELOG.md).
+
+<!--
+(arrow_dev) bkmgit@Linux:~/arrow-rs$ git log --pretty=oneline 5.0.0..6.0.0 | wc -l
+     99
+(arrow_dev) bkmgit@Linux:~/arrow-rs$ git shortlog -sn 5.0.0..6.0.0 | wc -l
+     35
+-->
+
+The Rust Arrow implementation would not be possible without the wonderful work and support of our community, and 
+the 6.0.0 release is no exception. It includes 99 commits from 35 individual contributors, many of them with 
+their first contribution. Thank you all very much.
+
+# Arrow
+
+Highlighted features and changes between release 5.0.0 and this release are:
+1. New MapArray support
+2. Add optimized filter kernel for regular expression matching
+3. Support arrow map datatype
+4. Implement sort() for binary array
+5. Replace `ArrayData::new()` with `ArrayData::try_new()` and `unsafe ArrayData::new_unchecked`
+6. Sorting should require less memory and be faster
+
+We continue to leverage the Rust ecosystem to deliver reliable and performant code.
+
+Of course, this release also contains bug fixes, performance improvements, and improved documentation examples. For the full list of changes, 
+please consult the [changelog](https://github.com/apache/arrow-rs/blob/6.0.0/CHANGELOG.md).
+
+# More Frequent Releases
+Arrow releases major versions every three months. The Rust implementation has been experimenting with releasing minor version updates to 
+speed the flow of new features and fixes.
+
+You can always find the latest releases on crates.io: [`arrow`](https://crates.io/crates/arrow), [`parquet`](https://crates.io/crates/parquet), 
+[`arrow-flight`](https://crates.io/crates/arrow-flight), and [`parquet-derive`](https://crates.io/crates/parquet-derive).
+
+# DataFusion & Ballista
+[DataFusion](https://docs.rs/datafusion/4.0.0/datafusion/) is an in-memory query engine with DataFrame and SQL APIs, built on top of Arrow. 
```

Comment:
```suggestion
[DataFusion](https://docs.rs/datafusion/) is an in-memory query engine with DataFrame and SQL APIs, built on top of Arrow. 
```

Reply:
How about using this doc link: https://arrow.apache.org/datafusion/